u8timeseries/metrics/metrics.py

```diff
@@ -28,23 +37,49 @@ def mape(true_series: TimeSeries, pred_series: TimeSeries) -> float:
     return 100. * np.mean(np.abs((y_true - y_hat) / y_true))
 
 
-def mase(true_series: TimeSeries, pred_series: TimeSeries) -> float:
+def mase_old(true_series: TimeSeries, pred_series: TimeSeries) -> float:
     """
-    Computes the Mean Average Scaled Error (MASE).
+    Computes the Mean Absolute Scaled Error (MASE).
 
     This function computes the MASE of `pred_series` with respect to `true_series`.
 
     :param true_series: A TimeSeries.
     :param pred_series: A TimeSeries to be compared with `true_series`.
     :return: A float, the MASE of `pred_series` with respect to `true_series`.
     """
+    warn("This function does not take into account the seasonality of the time series. "
+         "Please use mase_seasonal to be accurate", FutureWarning)
     y_true, y_pred = _get_values_or_raise(true_series, pred_series)
     errors = np.sum(np.abs(y_true - y_pred))
     t = y_true.size
     scale = t/(t-1) * np.sum(np.abs(np.diff(y_true)))
     return errors / scale
 
 
+def mase_seasonal(true_series: TimeSeries, pred_series: TimeSeries, m: int = 1) -> float:
```

Comment:
really cool! Since the seasonal version is equivalent to the non-seasonal one with `m=1`, how about renaming this method to `mase` ?

Reply:
I have no problem with it. It was just to show the difference with the previous method